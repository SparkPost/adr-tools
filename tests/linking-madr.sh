adr init docs madr
adr new First Record
adr new Second Record
adr new Third Record
adr link 4 Amends 2 "Amended by"
adr link 4 Clarifies 3 "Clarified by"
head -12 docs/0002-first-record.md
head -12 docs/0003-second-record.md
head -14 docs/0004-third-record.md
