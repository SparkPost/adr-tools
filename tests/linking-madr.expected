adr init docs madr
docs/0001-record-architecture-decisions.md
adr new First Record
docs/0002-first-record.md
adr new Second Record
docs/0003-second-record.md
adr new Third Record
docs/0004-third-record.md
adr link 4 Amends 2 "Amended by"
adr link 4 Clarifies 3 "Clarified by"
head -12 docs/0002-first-record.md
# *[short title of solved problem and solution]*
 
| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Amended by [ *[short title of solved problem and solution]* ](0004-third-record.md) |

**User Story:** *[ticket/issue-number]* <!-- optional -->

*[context and problem statement]*
*[decision drivers | forces]* <!-- optional -->

head -12 docs/0003-second-record.md
# *[short title of solved problem and solution]*
 
| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Clarified by [ *[short title of solved problem and solution]* ](0004-third-record.md) |

**User Story:** *[ticket/issue-number]* <!-- optional -->

*[context and problem statement]*
*[decision drivers | forces]* <!-- optional -->

head -14 docs/0004-third-record.md
# *[short title of solved problem and solution]*
 
| Date | Status |
| -- | -- |
| 1992-01-12 | Accepted |
| 1992-01-12 | Amends [ *[short title of solved problem and solution]* ](0002-first-record.md) |
| 1992-01-12 | Clarifies [ *[short title of solved problem and solution]* ](0003-second-record.md) |

**User Story:** *[ticket/issue-number]* <!-- optional -->

*[context and problem statement]*
*[decision drivers | forces]* <!-- optional -->

## Considered Alternatives
